db/migrate/20180424141617_azure_backslash_to_forward_slash.rb

```diff
@@ -0,0 +1,75 @@
+class AzureBackslashToForwardSlash < ActiveRecord::Migration[5.0]
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStack < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStackOutput < ActiveRecord::Base
+    belongs_to :stack, :class_name => 'OrchestrationStack'
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStackParameter < ActiveRecord::Base
+    belongs_to :stack, :class_name => 'OrchestrationStack'
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStackResource < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStackTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    Vm.where(:type => "ManageIQ::Providers::Azure::CloudManager::Vm").each do |vm|
+      vm.ems_ref = vm.ems_ref.tr("\\", '/')
+      vm.uid_ems = vm.ems_ref.tr("\\", '/')
+      vm.description = vm.description.tr("\\", '/')
```

Comment:
I think we can do all of this with a pure SQL query, which might save a lot of iterations and cycles... see https://www.postgresql.org/docs/9.5/static/functions-string.html